modules/admin_manual/pages/configuration/server/occ_commands/app_commands/richdocuments.adoc

```diff
@@ -0,0 +1,84 @@
+xref:enterprise/collaboration/collabora_secure_view.adoc[Collabora Online / Secure View] documentation.
+
+`config:app` commands configure the Collabora Online app.
+
+[source,console]
+----
+ config:app:get|set
+ richdocuments wopi_url            Number of days passed after upload (or restore)
+                                   that files will be archived
+ richdocuments secure_view_option  Number of days passed after archiving that files will
+                                   be permanently deleted
+ richdocuments watermark_text      Define groups of users that are exempt from the
+                                   Lifecycle policies
+ richdocuments secure_view_open_action_default  Restoration Policies for Users
+----
+
+== App Configuration
+
+All app configurations are set and queried with the `config:app` command set. The examples below set a value. To query a value use `config:app:get` and the corresponding key without any options or attributes.
+
+== Set the WOPI URL
+
+The number of days passed after upload (or restore) that files will be archived.
+
+The following example command sets the time passed since upload (or restore) for archiving files to 90 days.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set richdocuments wopi_url --value='[your-host-public-ip]:8098'
+----
+
+== Set the secure_view_option
+
+The number of days passed after archiving that files will be permanently deleted.
+
+The following example command sets the time passed to delete files to 180 days.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set richdocuments secure_view_option --value='true'
+----
+
+== Set the watermark_text
+
+Define groups of users that are exempt from the lifecycle policies (comma-separated group ids).
+
+The following example command sets the groups which members will not be part of the lifecycle management.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set richdocuments watermark_text --value="Restricted to {viewer-email}"
+----
+
+== Set the secure_view_open_action_default
+
+Set a policy who can restore files. Use the value `soft` for self-service and `hard` for admin/groupadmin-service.
+
+The following example command sets the restauration policy for users to `soft` (default).
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set richdocuments secure_view_open_action_default --value='true'
+----
+
+== Delete a Key or Change a Key
+
+You can delete a key with following example command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:delete richdocuments secure_view_open_action_default
+----
+
+You can change a key with following example command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set richdocuments secure_view_option --value='false'
+----
```

Comment:
Needs a whole refactor, not quite correct

Reply:
> not quite correct

ok, pls make suggestions.
(I have to take information from the PR merged from richdocuments and do my best from that what stands there)